MAUI/PDF-Viewer/Toolbar.md

```diff
@@ -109,14 +109,6 @@ The names of these toolbars and its description are listed in the following sect
 <td>StickyNoteIconsToolbar</td>
```

Comment:
Is this cross checked with source for any missed items?

Reply:
Yes, In mobile Totally 20 toolbar is available in the Toolbar list which include this one

Reply:
1. All 20 valid mobile toolbars are listed?
2. Cross checked desktop toolbars?

Reply:
<img width="457" alt="image" src="https://github.com/user-attachments/assets/63cb1480-1e14-498a-a38d-087245c16a60">


https://teams.microsoft.com/l/message/19:2ae5d162462848baa03c10765de62071@thread.skype/1725520126693?tenantId=77f1fe12-b049-4919-8c50-9fb41e5bb63b&groupId=f3c90ad3-a5ec-4dbe-9e9a-c2e11b9272f7&parentMessageId=1725511771695&teamName=File%20Formats%20%26%20Diagram%20Team&channelName=MAUI%20PDF%20Viewer&createdTime=1725520126693

Reply:
1.All 20 valid mobile toolbars are listed?
Yes I have verified by checking it names.

2.Cross checked desktop toolbars?
Desktop has 17 toolbars, It also verfied